.pre-commit-config.yaml

```
@@ -61,4 +61,4 @@ repos:
         name: integrity
         entry: 'npm run integrity --force'
         language: node
-        stages: [push]
+        stages: [Nonepre-pushNone]
```

This is a bug upstream, already fixed https://github.com/pre-commit-ci/issues/issues/234

```suggestion
        stages: [pre-push]
```